packages/flutter_tools/test/general.shard/tester/flutter_tester_test.dart

```diff
@@ -187,8 +189,19 @@ FlutterTesterDevices setUpFlutterTesterDevices() {
       logger: logger,
     ),
     flutterVersion: MockFlutterVersion(),
+    operatingSystemUtils: MockOperatingSystemUtils(),
   );
 }
 
 class MockBuildSystem extends Mock implements BuildSystem {}
 class MockFlutterVersion extends Mock implements FlutterVersion {}
+class MockOperatingSystemUtils extends Mock implements OperatingSystemUtils {
```

Comment:
Sorry, but can you re-write this with `FakeOperatingSystemUtils`? As part of https://github.com/flutter/flutter/issues/71511, we're trying to remove all code that extends `Mock`, as it is not null-safe.

Reply:
Sorry, I fixed unit tests.